horreum-backend/src/main/java/io/hyperfoil/tools/horreum/svc/TestServiceImpl.java

```diff
@@ -257,7 +257,7 @@ public List<Test> list(String roles, Integer limit, Integer page, String sort, S
       if (limit != null && page != null) {
          query.page(Page.of(page, limit));
       }
-      return query.list().stream().map(TestMapper::from).collect(Collectors.toList());
+      return new TestQueryResult query.list().stream().map(TestMapper::from).collect(Collectors.toList()). Test.count();
```

Comment:
This does not compile, needs to be 

```suggestion
      return new TestQueryResult( query.list().stream().map(TestMapper::from).collect(Collectors.toList()), Test.count() ) ;
```